lib/v2/chinacef/index.js

```diff
@@ -0,0 +1,68 @@
+const utils = require('./utils');
+const cheerio = require('cheerio');
+const got = require('@/utils/got');
+const parseDate = require('@/utils/dateParser');
```

Comment:
Please use `parseDate` from `@/utils/dateParser`

Reply:
I don't get here. I already used the `parseDate` from `@/utils/dateParser`.

Reply:
My mistake. It should be `@/utils/parse-date`.